elections57_results.py

```diff
@@ -223,16 +225,19 @@ def get_class(v):
     from datetime import datetime
 
     def parse_timestamp(s):
-        return datetime.strptime(s, '%m/%d/%Y %H:%M:%S')
+        return datetime.strptime(s, '%Y-%m-%d %H:%M:%S')
+
+    # See http://stackoverflow.com/questions/6832445/how-can-bcrypt-have-built-in-salts
+    salt = bcrypt.gensalt()
```

Comment:
Factor is a good idea, thank. Re: calling `hashpw()` every time - that was my original idea, before commit 93edf91 (and you can test it), but I suspect it will produce different hashes for the same person, and therefore the person who has just the output file won't be able to confirm which votes have been cast by the same person.